### ISSUE TITLE
Inconsistent use of extensions with main.ts and main.ns.ts

### ISSUE THREAD

**Issue by tjvantoll** [NativeScript/nativescript-schematics#102]
I’m writing a tutorial on code sharing with this setup, and I found a weird inconsistency.

For most files in the app we use a `.tns` extension to denote {N} files. But strangely, in the root `src` folder we create `main.ns.aot.ts` and `main.ns.ts` files. Can those files use the same `.tns` extension? It’s a strange inconsistency to explain.

**Comment by sis0k0** [comment]
Thank you for the feedback, TJ!

The `main.aot.ts` file is not needed anymore. Instead, we introduced a build-time TypeScript transformer. It converts the JiT bootstrap code to AoT bootstrap code.

As for the `main.ns.ts` file - indeed, there is a reason to have a weird suffix (`ns.ts` instead of `tns.ts`).
The short version:
- The files with a `.tns.ts` suffix shouldn't be part of the TS compilation for mobile. They are excluded with a glob - `**/*.tns.ts`.
- The entry file should be part of the TS compilation for mobile. That's why it's named with a different suffix - `main.ns.ts`.

The good news:
I've read the tsconfig documentation more carefully :rofl: . I've found a way to rename the file to `main.tns.ts` and include it in the mobile build.
